app/assets/stylesheets/common/_markus.scss

```diff
@@ -1004,6 +1004,7 @@ nav {
     position: absolute;
     text-align: left;
     top: 1.95em;
+    width: 101%;
```

Comment:
So rather than do this, I think it's okay to change the `min-width` setting to be the same as the parent's. (I believe changing that will also allow the dropdown to be *wider* than the parent if the text is longer, though to be honest I'm not sure. Perhaps you could try it?)

Reply:
I found that for both, if a single word is short than about 33 characters the new words will be on the next line if they cannot fit on the existing line. When this is not the case for a single word, min-width does make the dropdown box larger to fit and width makes the box the same size and the test goes outside to the right of it.

Reply:
@therealfake I'm having trouble picturing what you're describing. Can you attach screenshots with this rule toggled on/off to illustrate what you mean?

Reply:
@david-yz-liu Sure!
So this image shows when each word itself is < ~25 chars it wraps to the next line and the box is unaffected:
![image](https://user-images.githubusercontent.com/76544772/157361442-be840844-9e5e-4dc8-961b-9c43ee07b139.png)
This is what happens when we just go with ```width: 101%```:
(The word is "AboutMarkUsAboutMarkUsAboutMarkUs" which is 33 chars)
![image](https://user-images.githubusercontent.com/76544772/157361758-420c736c-9223-4792-96ed-e59305e93fec.png)
And this is what happens when we just go with ```min-width: 101%```:
![image](https://user-images.githubusercontent.com/76544772/157362139-7f72bd34-f274-4262-844d-fd9e05b38c14.png)